examples/join-tables-and-streams/join_tables_and_streams.bal

```diff
@@ -0,0 +1,88 @@
+import ballerina/io;
+import ballerina/runtime;
+
+//This is the record that holds item details in the stockTable.
+type Item record {
+    string name;
+    float price;
+    int stockAmount;
+};
+
+//This is the record that holds order events from customer.
+type Order record {
+    string itemName;
+    int orderingAmount;
+};
+
+//This is the record that holds alert events.
+type OutOfStockAlert record {
+    string itemName;
+    int stockAmount;
+};
```

Comment:
It would be nice to have the records closed if you don't expect more items in the record.

Reply:
done